pom.xml

```
@@ -79,8 +79,8 @@
         <dom4j.version>2.1.3</dom4j.version>
         <h2.version>2.1.214</h2.version>
         <jakarta.persistence.version>2.2.3</jakarta.persistence.version>
-        <hibernate.core.version>5.6.10.Final</hibernate.core.version>
-        <hibernate.c3p0.version>5.6.10.Final</hibernate.c3p0.version>
+        <hibernate-core.version>5.6.14.Final</hibernate-core.version>
```

Should we rename it even more to [align with Quarkus](https://github.com/quarkusio/quarkus/blob/2.13.5.Final/bom/application/pom.xml#L91)?
```suggestion
        <hibernate-orm.version>5.6.14.Final</hibernate-core.version>
```

---

Thanks, I addressed it in a34d0f7.